From 27e66d7da858bc012ea390685ab3be2373dc347a Mon Sep 17 00:00:00 2001
From: Stephan Linz <linz@li-pro.net>
Date: Sun, 16 Jan 2011 08:51:33 +0100
Subject: [PATCH 06/10] uio_of_genirq: fix incompatible pointer type

drivers/uio/uio_of_genirq.c:41: warning: passing argument 3 of '__uio_pdrv_genirq_probe' from incompatible pointer type

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 drivers/uio/uio_of_genirq.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/uio/uio_of_genirq.c b/drivers/uio/uio_of_genirq.c
index 254ec5b..b6ce382 100644
--- a/drivers/uio/uio_of_genirq.c
+++ b/drivers/uio/uio_of_genirq.c
@@ -38,7 +38,7 @@ static __devinit int uio_of_genirq_probe(struct of_device *op,
 		if (of_address_to_resource(op->node, i, &resources[i]))
 			break;
 
-	ret = __uio_pdrv_genirq_probe(&op->dev, uioinfo, &resources, i);
+	ret = __uio_pdrv_genirq_probe(&op->dev, uioinfo, resources, i);
 	if (ret)
 		goto err_cleanup;
 
-- 
1.6.0.4

